translations/messages+intl-icu.lt.xlf

```diff
@@ -0,0 +1,430 @@
+            </trans-unit>
+            <trans-unit id="post.search_no_results">
+                <source>post.search_no_results</source>
+                <target>Nieko nerasta</target>
+            </trans-unit>
+
+            <trans-unit id="user.updated_successfully">
+                <source>user.updated_successfully</source>
+                <target>Vartotojas atnaujintas sėkmingai!</target>
+            </trans-unit>
+
+            <trans-unit id="notification.comment_created">
+                <source>notification.comment_created</source>
+                <target>Jūsų įrašas gavo komentarą!</target>
+            </trans-unit>
+            <trans-unit id="notification.comment_created.description">
+                <source>notification.comment_created.description</source>
+                <target><![CDATA[Jūsų įrašas "{title}" gavo naują komentarą. Kad peržiūrėti komentarą, galite paspausti šia <a href="{link}">nuorodą</a>]]></target>
+            </trans-unit>
+
+            <trans-unit id="help.app_description">
+                <source>help.app_description</source>
+                <target><![CDATA[Tai yra <strong>demo puslapis</strong> sukurtas su Symfony Framework parodydant tinkamą Symfony panaudojimą.]]></target>
+            </trans-unit>
+            <trans-unit id="help.show_code">
+                <source>help.show_code</source>
+                <target><![CDATA[Paspauskite mygtuką, kad peržiūrėti <strong>Controller</strong> ir <strong>šabloną</strong> panaudoti atvaizduojant šį puslapį.]]></target>
+            </trans-unit>
+            <trans-unit id="help.browse_app">
+                <source>help.browse_app</source>
+                <target><![CDATA[Naršyti <strong>public dalį</strong> demo puslapio.]]></target>
+            </trans-unit>
+            <trans-unit id="help.browse_admin">
+                <source>help.browse_admin</source>
+                <target><![CDATA[Naršyti <strong>admin dalį</strong> demo puslapio.]]></target>
+            </trans-unit>
+            <trans-unit id="help.login_users">
+                <source>help.login_users</source>
+                <target>išbandykite vieną iš šių vartotojų</target>
+            </trans-unit>
+            <trans-unit id="help.role_user">
+                <source>help.role_user</source>
+                <target>nuolatinis vartotojas</target>
+            </trans-unit>
+            <trans-unit id="help.role_admin">
+                <source>help.role_admin</source>
+                <target>administratorius</target>
+            </trans-unit>
+            <trans-unit id="help.reload_fixtures">
+                <source>help.reload_fixtures</source>
+                <target>Jei vartotojai neveikia, perleiskite šia komandą:</target>
+            </trans-unit>
+            <trans-unit id="help.add_user">
+                <source>help.add_user</source>
+                <target>Jeigu norite sukurti naują vartotoją paleiskite šia komandą:</target>
```

Comment:
Probably a false positive, but Google Translate says that `Jeigu` -> should be -> `Jei`